core/execution/evm/mod.go

```diff
@@ -46,12 +49,13 @@ var txParams = struct {
 	// amount of Gwei (nano ether) that the user is willing to spend on each
 	// unit of Gas
 	GasPrice *big.Int
-}{1e7, big.NewInt(1)}
+}{6721975, big.NewInt(20)}
```

Comment:
Since this piece is pretty important to make the code work I would add a small comment about the value, where it comes from and why it must be like so.

Reply:
Added a comment